src/web_app_skeleton/script/check_system

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+BOLD_RED_COLOR="\e[1m\e[31m"
+
+# Prints error and exit.
+# example:
+#   print_error "Redis is not running. Run it with some_command"
+print_error() {
+    printf "${BOLD_RED_COLOR}There is a problem with your system setup:\n\n   ${BOLD_RED_COLOR}$1 \n\n"
+    exit 1
+}
+
+# Use this script to add any processes required to boot your application.
```

Comment:
```suggestion
# Use this script to check the system for required tools and process that your app needs.
```

Since this could also be used to do generic checks like crystal version, yarn installed, etc.